src/plots/cartesian/constants.js

```diff
@@ -0,0 +1,41 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+module.exports = {
+    // ms between first mousedown and 2nd mouseup to constitute dblclick...
+    // we don't seem to have access to the system setting
+    DBLCLICKDELAY: 600,
+
+    // pixels to move mouse before you stop clamping to starting point
+    MINDRAG: 8,
```

Comment:
Would be a bit more code, but making the `MINDRAG` a function of the main drag distance would be useful I think. Something with a log scale would be useful. If I 1D drag the width of my screen, it's likely going to fluctuate a whole lot more than if I drag 100px.